io.lmms.LMMS.appdata.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2019 the authors -->
+<component type="desktop">
+ <id>io.lmms.LMMS</id>
+ <metadata_license>CC0</metadata_license>
+ <project_license>GPL-2.0+</project_license>
+ <name>LMMS</name>
+ <summary>A music production application</summary>
+ <description>
+   <p>LMMS is a music production application. It allows the creation
+   of loops, synthesizing and mixing sounds, arranging samples having
+   fun with your MIDI-keyboard and much more... It combines the
+   features of a tracker-/sequencer-program and those of powerful
+   synthesizers, samplers, effects etc. in a modern, user-friendly and
+   easy to use graphical user-interface.</p>
+ </description>
+ <content_rating type="oars-1.1" />
+ <launchable type="desktop-id">io.lmms.LMMS.desktop</launchable>
+ <translation type="qt">lmms</translation>
+ <screenshots>
+  <screenshot type="default">
+   <image>https://lmms.io/wiki/images/thumb/2/2d/Startup_LMMS_new.png/1200px-Startup_LMMS_new.png</image>
+   <caption>The main LMMS window</caption>
+  </screenshot>
+ </screenshots>
+ <url type="homepage">https://lmms.io/</url>
+ <releases>
+   <release version="1.2.0-rc7" />
```

Comment:
@hfiguiere 

> OK, the last changes are in. I'll see about upstreaming the appdata.

You need to specify date as well.
https://www.freedesktop.org/software/appstream/docs/chap-Metadata.html#tag-releases

Next time try to validate it before you make a commit.
```
$ appstream-util validate-relax --nonet io.lmms.LMMS.appdata.xml 
io.lmms.LMMS.appdata.xml: FAILED:
• attribute-missing     : <release> has no timestamp
Validation of files failed
```

Also, it would be nice if you put release notes like here:
https://github.com/scx/ocrfeeder-flatpak/blob/26e656579d3dca899b36d4ba1dade65b57abbb32/ocrfeeder.appdata.xml#L174-L190

You can find them here:
https://lmms.io/download/#linux

See also:
https://www.freedesktop.org/software/appstream/docs/chap-Metadata.html#tag-releases
https://www.freedesktop.org/software/appstream/docs/chap-Metadata.html#tag-description

Reply:
done. sorry I forgot to run the validator.